src/js/player.js

```diff
@@ -2848,6 +2850,10 @@ class Player extends Component {
     const tagOptions = Dom.getElAttributes(tag);
     const dataSetup = tagOptions['data-setup'];
 
+    if (Dom.hasElClass(tag, 'vjs-fluid')) {
```

Comment:
should we add a test that verifies that if `vjs-fluid` class is available then `player.fluid()` returns `true`?